Easydict/objc/ViewController/Window/BaseQueryWindow/EZBaseQueryWindow.m

```diff
@@ -128,7 +128,7 @@ - (void)windowDidResignKey:(NSNotification *)notification {
 }
 
 - (void)windowDidResize:(NSNotification *)aNotification {
-    NSLog(@"windowDidResize: %@, windowType: %ld", @(self.frame), self.windowType);
+//    NSLog(@"windowDidResize: %@, windowType: %ld", @(self.frame), self.windowType);
```

Comment:
I saw some places where we are using `print` and `NSLog` to print some info,  maybe we could consider using a uniform log method later, we already import packages like `CocoaLumberjack` and `swift-log`.

Reply:
Very good idea!

`swift-log` looks like a library that comes with `CocoaLumberjack`, but I'm not very familiar with `CocoaLumberjack`, and I'm confused about the relationship between them.

We actually have a legacy logging module in our project, `MMLog`, which provides useful crash logs when users experience crashes.

However, I think the `MMLog` code is not very good, it uses a lot of macros, looks complicated, and is not easy to use with external APIs, e.g., it doesn't distinguish well between normal logs, file logs, warning logs, etc. In addition, it uses some deprecated methods, resulting in having to use `#pragma clang diagnostic ignored` to ignore warnings.

I've thought about optimizing this part of the code before, but I'm not familiar with `CocoaLumberjack` and modifying it would be too difficult for me.

If you are familiar with log, can you help optimize this code?

Reply:
Sure, I checked the documentation, and it seems pretty easy to use. Replacing the existing log should not take long.